telegram/ext/_jobqueue.py

```diff
@@ -447,6 +453,11 @@ async def callback(context: CallbackContext)
             queue.
 
         """
+        warn(
+            "Prior to v20.0 the `days` parameter was not aligned to that of cron's weekday scheme."
+            "We recommend double checking if the passed value is correct.",
+            stacklevel=2,
+        )
```

Comment:
What about a comment here, reminding us that we remove this in the next major version?

Reply:
is this enough?
```python
        # TODO: After v20.0, we should remove the this warning.
```

Reply:
should be enough IMO, yes :)